projects/js-packages/critical-css-gen/package.json

```
@@ -0,0 +1,91 @@
+	"type": "module",
+	"name": "@automattic/jetpack-critical-css-gen",
+	"version": "0.1.0-alpha",
+	"description": "A flexible Critical CSS Generator that supports multiple URLs and viewports, with both server-side and client-side generation capabilities.",
+	"homepage": "https://github.com/Automattic/jetpack/tree/HEAD/projects/js-packages/critical-css-gen/#readme",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/labels/[JS Package] Critical Css Gen"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git",
+		"directory": "projects/js-packages/critical-css-gen"
+	},
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic",
+	"scripts": {
+		"build:browser": "rollup -c",
+		"build:node": "tsc --project tsconfig.node.json",
+		"build": "npm run clean && npm run build:browser && npm run build:node",
+		"clean": "rm -rf lib/ && rm -rf dist/",
+		"lint": "prettier --check src/**/* && eslint . && echo '✔ eslint and prettier ran successfully.'",
+		"lint:fix": "prettier --write src/**/*.ts && eslint --fix . && echo '✔ eslint and prettier ran successfully.'",
+		"test": "npm run build && NODE_ENV=test NODE_PATH=./node_modules jest --forceExit --config=tests/config/jest.config.js"
+	},
+	"main": "./lib/back-end.js",
+	"browser": "./dist/bundle.js",
+	"devDependencies": {
+		"@babel/core": "7.24.7",
+		"@babel/preset-env": "7.24.7",
+		"@babel/preset-typescript": "7.24.7",
+		"@rollup/plugin-commonjs": "26.0.1",
+		"@rollup/plugin-json": "4.1.0",
+		"@rollup/plugin-node-resolve": "13.3.0",
+		"@rollup/plugin-terser": "0.4.3",
+		"@rollup/plugin-typescript": "8.3.3",
+		"@types/clean-css": "4.2.5",
+		"@types/css-tree": "2.0.1",
+		"@types/node": "^20.4.2",
+		"@typescript-eslint/eslint-plugin": "6.21.0",
+		"@typescript-eslint/parser": "6.21.0",
+		"eslint": "8.57.0",
+		"express": "4.18.1",
+		"https-browserify": "1.0.0",
+		"jest": "29.7.0",
+		"jest-environment-puppeteer": "10.0.1",
+		"node-fetch": "3.3.2",
+		"nodemon": "3.1.2",
+		"os-browserify": "0.3.0",
+		"path-browserify": "1.0.1",
+		"playwright-core": "1.33.0",
+		"prettier": "npm:wp-prettier@2.0.5",
+		"puppeteer": "22.8.2",
+		"puppeteer-core": "22.6.5",
+		"rollup": "2.78.1",
+		"rollup-plugin-polyfill-node": "0.10.2",
+		"source-map": "0.7.4",
+		"source-map-js": "1.2.0",
+		"stream-http": "3.2.0",
+		"tslib": "2.5.0",
+		"typescript": "5.0.4",
+		"webpack": "5.76.0",
+		"webpack-dev-middleware": "5.3.3"
+	},
+	"exports": {
+		".": {
+			"jetpack:src": "./src/back-end.ts",
+			"types": "./lib/back-end.d.ts",
+			"browser": "./dist/bundle.js",
+			"import": "./lib/back-end.js",
+			"require": "./lib/back-end.js",
+			"default": "./lib/back-end.js"
+		}
+	},
+	"dependencies": {
+		"clean-css": "5.3.1",
+		"css-tree": "2.3.1"
+	},
+	"peerDependencies": {
+		"puppeteer-core": "22.6.5"
+	},
+	"peerDependenciesMeta": {
+		"puppeteer-core": {
+			"optional": true
+		}
+	},
+	"overrides": {
+		"eslint-plugin-jsdoc": "46.8.2"
+	}
```

This doesn't do anything in pnpm, AFAIK.
```suggestion
	}
```

---

Updated in a0e29e7f7d8a0ddb3f78412a8c875f9a2525eb09.